P/polymake/bundled/config/config-x86_64-apple-darwin14.ninja

```diff
@@ -0,0 +1,66 @@
+# last configured with:
+configure.command=/workspace/srcdir/polymake/configure CXX=clang++ --with-libcxx --with-lrs-include=/workspace/destdir/include/lrslib
+root=/workspace/srcdir/polymake
+core.includes=-I${root}/include/core-wrappers -I${root}/include/core
+app.includes=-I${root}/include/app-wrappers -I${root}/include/apps -I${root}/include/external/permlib -I${root}/include/external/TOSimplex
+
+CC = clang -target x86_64-apple-darwin14 --sysroot /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root
+CXX = clang++ -target x86_64-apple-darwin14 --sysroot /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root
+CFLAGS = -mmacosx-version-min=10.8 -I/workspace/destdir/deps/GMP_jll/include -I/workspace/destdir/deps/MPFR_jll/include
+CXXFLAGS = -stdlib=libc++ -std=c++14 -mmacosx-version-min=10.8 -I/workspace/destdir/deps/GMP_jll/include -I/workspace/destdir/deps/MPFR_jll/include -I/workspace/destdir/deps/boost_jll/include -I/workspace/destdir/deps/FLINT_jll/include -DPOLYMAKE_WITH_FLINT
+ARCHFLAGS = 
+CsharedFLAGS = -fPIC -pipe
+CXXOPT = -O3
+CXXDEBUG = -g
+CXXCOV = --coverage -O1
+CXXSANITIZE = -fno-omit-frame-pointer -O1 -g
+CflagsSuppressWarnings = 
+CLANGversion = 4.0
+CPPStd = 201402
+XcodeVersion = 9.1.0
+LDFLAGS = -mmacosx-version-min=10.8 -L/workspace/destdir/deps/GMP_jll/lib -Wl,-rpath,/workspace/destdir/deps/GMP_jll/lib -L/workspace/destdir/deps/MPFR_jll/lib -Wl,-rpath,/workspace/destdir/deps/MPFR_jll/lib -L/workspace/destdir/deps/FLINT_jll/lib -Wl,-rpath,/workspace/destdir/deps/FLINT_jll/lib -stdlib=libc++
+LDsharedFLAGS = -mmacosx-version-min=10.8 -L/workspace/destdir/lib -Wl,-rpath,/workspace/destdir/lib -dynamiclib -undefined dynamic_lookup -fstack-protector-strong
+LDcallableFLAGS = -mmacosx-version-min=10.8  -L/workspace/destdir/lib -Wl,-rpath,/workspace/destdir/lib -dynamiclib -undefined dynamic_lookup -fstack-protector-strong  -mmacosx-version-min=10.8 -fstack-protector-strong
+LDsonameFLAGS = -install_name /workspace/destdir/lib/
+LIBS =  -lc++ -lflint -lmpfr -lgmp -lpthread
+ExternalHeaders =  permlib TOSimplex
+Arch = darwin.x86_64
+BundledExts = cdd atint bliss flint libnormaliz lrs ppl sympol
+BuildModes = Opt Debug
+InstallTop = /workspace/destdir/share/polymake
+InstallArch = /workspace/destdir/lib/polymake
+InstallBin = /workspace/destdir/bin
+InstallInc = /workspace/destdir/include
+InstallLib = /workspace/destdir/lib
+InstallDoc = /workspace/destdir/share/polymake/doc
+AR = ar
+GMP.version = 6.1.2
+MPFR.version = 4.0.2
```

Comment:
Will this still work when the latest version of the corresponding JLLs (6.2.0 and 4.1.0) are installed?

Reply:
In polymake these variables are not really used (except for some testcases). But I don't really know how binary-compatible these GMP versions are, especially when building against GMP 6.1.2 and then using 6.2.0.

I see three options:
- It will just work (GMP 6.2 and 6.1.2 have the same soname at least on linux, but there were some related issues on MacOS?)
- Maybe it will work when it is built with GMP 6.2.
- We need to provide different binaries depending on the julia / GMP version, but I don't know if this is feasible.

Reply:
Anything compiled against GMP 6.1.2 should work fine with GMP 6.2 provided it uses only the official APIs. Of course things that directly poke at internals can always run into problems; but even those are fine, with one exception: GMP 6.2 introduced an optimization where the storage pointer in an `mpz_t` can be initialized lazily, i.e., it can be NULL; I think some code in FLINT tripped over this at some point (but that's fixed in FLINT 2.6.2).

So I think we should be fine. I really just was wondering if Polymake would be checking those versions somewhere and refuse to compile and/or run if they differ.